inspire/modules/workflows/tasks/submission.py

```diff
@@ -256,3 +256,15 @@ def add_files_to_task_results(obj, eng):
         obj.add_task_result(file_obj.name,
                             fileinfo,
                             "workflows/results/files.html")
+
+
+def add_note_entry(obj, eng):
+    """Add metadata to sip on approval."""
+    if was_approved(obj, eng):
+        rt = get_instance() if cfg.get("PRODUCTION_MODE") else None
```

Comment:
I guess this `rt` thing is here as it is `wip`? I don't think we need rt in this context.

If you wanted to create a RT curation ticket tho, I'd suggest creating a new task for it - just to separate it from the note manipulation.

Come to think it of, since there is so much CORE record checking, we could evaluate if we should add another `workflow_if` etc branch in the workflow itself where we do in one case all the core business and `workflow_else` for all non-core business. Or is it overkill? WDYT?